.github/workflows/documentation.yml

```diff
@@ -84,22 +88,13 @@ jobs:
         echo "using xsltproc ;" >> $HOME/user-config.jam
         test -f $HOME/user-config.jam && cat $HOME/user-config.jam
 
-    - name: Install libs/geometry/doc/src/docutils/tools/doxygen_xml2qbk
-      run: |
-        cd $BOOST_ROOT
-        $BOOST_ROOT/b2 variant=release libs/geometry/doc/src/docutils/tools/doxygen_xml2qbk
-        test -f /usr/local/bin/doxygen_xml2qbk && rm -rf /usr/local/bin/doxygen_xml2qbk
-        sudo cp $BOOST_ROOT/dist/bin/doxygen_xml2qbk /usr/local/bin/
-        ls -l /usr/local/bin/b2
-        doxygen_xml2qbk --version
-
-    - name: Build libs/geometry/doc/src/examples
+    - name: Build examples
       run: |
         cd $BOOST_ROOT
         $BOOST_ROOT/b2 libs/geometry/doc/src/examples
 
-    - name: Run libs/geometry/doc/make_qbk.py
+    - name: Build documentation
       run: |
         cd $BOOST_ROOT
         cd libs/geometry/doc
-        python ./make_qbk.py
+        $BOOST_ROOT/b2
```

Comment:
This is perfect! So we call `b2` now instead of `make_qbk.py`, cool.

Can we then remove the fact that `make_qbk.py` calls `b2`? 
Is everyone following this workflow?
Or do we, if we do that, disturb the making of the official release documentation?

Reply:
I do not know what scripts/workflow is used to build the official release documentation. @barendgehrels do you know?

Reply:
No, I don't know that exactly...